scripts/publish-root.gradle

```diff
@@ -0,0 +1,41 @@
+// Create variables with empty default values
+ext["signing.keyId"] = ''
+ext["signing.password"] = ''
+ext["signing.key"] = ''
+ext["ossrhUsername"] = ''
+ext["ossrhPassword"] = ''
+ext["sonatypeStagingProfileId"] = ''
+ext["snapshot"] = ''
+ext["PUBLISH_VERSION"] = ''
+
+File secretPropsFile = project.rootProject.file('local.properties')
+if (secretPropsFile.exists()) {
+    // Read local.properties file first if it exists
+    Properties p = new Properties()
+    new FileInputStream(secretPropsFile).withCloseable { is -> p.load(is) }
+    p.each { name, value -> ext[name] = value }
+} else {
```

Comment:
To run Gradle artifacts upload scripts via command instead of running Github workflow. Was very helpful to test and think might be helpful as well to debug issues related to scripts.